src/IssueViz.Security/OpenInIDE/OpenInIDEPackage.cs

```diff
@@ -33,7 +33,6 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE
     [PackageRegistration(UseManagedResourcesOnly = true, AllowsBackgroundLoading = true)]
     [Guid("5C38BF5B-F6D1-4232-A5B5-5664B6A19A91")]
     // We want to start listening as soon as the IDE starts.
-    // Note: in VS2019, this will be when the new modal start page appears.
```

Comment:
It was about vs17 to vs19. The info is still relevant. Change to Since vs19,...

Reply:
I think it's not that important since this is the default behaviour info about this change is not interesting anymore and I think it should've been removed when dropping 2017.

Reply:
I think it's valuable for someone to know that the shell initializes before you actually select any project, however this comment is not the most noticeable place, so up to you